src/components/KTreeList/KTreeList.cy.ts

```diff
@@ -0,0 +1,8 @@
+import { mount } from 'cypress/vue'
+import KTreeList from '@/components/KTreeList/KTreeList.vue'
+
+describe('KTreeList', () => {
```

Comment:
We definitely need tests before merge

Reply:
Agree, but I don't know how to test drag-n-drop and that is 90% of the code. I can test some of the props and slots, but looking for suggestions on the drag-n-drop

Reply:
Yeah maybe for starters you test everything that isn't related to drag-n-drop functionality? That will probably be it's own nested `describe` block in the tests file anyway